mappings/net/minecraft/entity/passive/VillagerEntity.mapping

```diff
@@ -1,20 +1,33 @@
 CLASS net/minecraft/unmapped/C_etbdktdp net/minecraft/entity/passive/VillagerEntity
 	FIELD f_ajxqvfcw lastGossipDecayTime J
+	FIELD f_bdqnpsun chasing Z
 	FIELD f_buldqrow ITEM_FOOD_VALUES Ljava/util/Map;
 	FIELD f_fxzlnzob levelingUp Z
 	FIELD f_godxqihz foodLevel B
 	FIELD f_hfjxwxmi gossipStartTime J
 	FIELD f_husfmqdc natural Z
+	FIELD f_hyllvadl GOLEM_SPAWN_REQUIRED_AGREEING_VILLAGERS I
+		COMMENT How many villagers should agree for a golem to be spawned.
 	FIELD f_iqnyhgsa lastCustomer Lnet/minecraft/unmapped/C_oqcyprzj;
+	FIELD f_iszvhyho SPEED_MODIFIER F
+	FIELD f_jhdzqprd TRADE_OFFERS_PER_LEVEL I
 	FIELD f_jzymtuba GATHERABLE_ITEMS Ljava/util/Set;
 	FIELD f_mxpcybbc lastRestockTime J
 	FIELD f_nsvjnijv gossip Lnet/minecraft/unmapped/C_vtagjako;
+	FIELD f_phlnrfen GOLEM_COMUNICATION_DISTANCE I
+		COMMENT How far away the villager should "talk" about golems with other villagers.
 	FIELD f_pwmlqhwj levelUpTimer I
 	FIELD f_rnmascnx VILLAGER_DATA Lnet/minecraft/unmapped/C_yxacmnir;
+	FIELD f_saemkfcb BREEDING_FOOD_REQUIRED I
+	FIELD f_setunyzz REPUTATION_CHANGE_PER_EVENT I
+	FIELD f_sluljwfk TIME_SINCE_SLEEPING_FOR_GOLEM_SPAWNING J
 	FIELD f_srqdypwf experience I
 	FIELD f_tdcxjuty restocksToday I
 	FIELD f_tejltula SENSORS Lcom/google/common/collect/ImmutableList;
 	FIELD f_wmwevlan POINTS_OF_INTEREST Ljava/util/Map;
```

Comment:
POINT_OF_INTEREST or is this a list of points of interest? If not don't name this points only point